paddle/fluid/operators/hierarchical_sigmoid_op.h

```diff
@@ -28,20 +30,46 @@ template <typename T, int MajorType = Eigen::RowMajor,
 using EigenMatrix = framework::EigenMatrix<T, MajorType, IndexType>;
 using platform::Transform;
 
+std::vector<int64_t> cal_rows(const framework::LoDTensor& path) {
```

Comment:
`cal_rows` -> `static ... cal_rows(...)`

Reply:
`cal_rows` --> `CalRows`

Also, `CalRows` is a really bad name.

Reply:
```cpp
static std::vector<int64_t> PathToRows(const framework::LoDTensor& path) {
  std::set<int64_t> rows;
  for (int64_t i=0;i<path.numel(); ++i) {
    int64_t row = path.data<int64_t >()[i];
    if (row < 0) {
      continue;
    }
    rows.emplace(row);
  }
  return std::vector<int64_t>(rows.begin(), rows.end());
}
```

Reply:
done